### ISSUE TITLE
Cannot stream videos from the Neos cloud

### ISSUE THREAD

**Issue by H3BO3** [Frooxius/NeosPublic#577]
Attempting to load/play a longer video file from the neos cloud in Stream mode throws a log error:

```
UnityVideoTexture Error: VideoPlayer cannot play url : https://cloudx.azureedge.net/assets/[hash]

Cannot read file.
```

Trying to access this url via browser gives "An error occurred while processing your request".

Replicated by taking a video player previously saved to the cloud for a file upload, switching it to stream mode, then re-saving and clearing cache and DB - Loading out the stream video player threw the error as I had seen previously.

**Comment by Frooxius** [comment]
Which videos do this? I have videos that stream from the cloud correctly.

**Comment by Frooxius** [comment]
Ok fixed in 2020.5.24.1361! It was due to the size of the files.
